src/components/CategoryNavContainer/CategoryNav/CategoryNav.js

```diff
@@ -1,34 +1,61 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import camelCase from 'lodash/camelCase';
 import { NavLink } from 'react-router-dom';
 import style from './CategoryNav.scss';
 import { NO_CATEGORY } from '../../../constants';
 
-const CategoryNav = props => (
-  <div>
-    <NavLink
-      className={style.link}
-      activeClassName={style.link__active}
-      to={`/${props.cityId}/${NO_CATEGORY}`}
-    >Overview</NavLink>
+class CategoryNav extends Component {
+  componentDidMount() {
+    this.scrollToActiveNavLink();
+  }
 
-    {props.categories.map(category => (
-      <NavLink
-        className={style.link}
-        activeClassName={style.link__active}
-        key={category.id}
-        to={`/${props.cityId}/${category.id}`}
-      > {category.name} </NavLink>
-    ))}
-  </div>
-);
+  componentDidUpdate() {
+    this.scrollToActiveNavLink();
+  }
+
+  scrollToActiveNavLink() {
+    if (this.activeLinkEl) {
+      this.el.scrollLeft = this.activeLinkEl.parentElement.offsetLeft - 20;
```

Comment:
IMO 40 looks better because you can see that there's more options off to the left.

Reply:
Deferring to Nick on this